### PR TITLE
[new release] lbfgs (0.9.3)

### PR DESCRIPTION
**Pull request by Chris00**
Bound-constrainted optimization in many variables

- Project page: <a href="https://github.com/Chris00/L-BFGS-ocaml">https://github.com/Chris00/L-BFGS-ocaml</a>
- Documentation: <a href="https://Chris00.github.io/L-BFGS-ocaml/doc">https://Chris00.github.io/L-BFGS-ocaml/doc</a>

##### CHANGES:

- Replace `camlp4` by `cppo`.

### CODE REVIEW

File: packages/lbfgs/lbfgs.0.9.3/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "@doc"] {with-doc}
+]
+depends: [
+  "ocaml"
+  "base-bigarray"
+  "base-bytes"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/lbfgs/lbfgs.0.9.3/opam]:
```suggestion
  "dune" {>= "1.1"}
```
According to your `dune-project` file this requires at least dune 1.1

File: packages/lbfgs/lbfgs.0.9.3/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+homepage: "https://github.com/Chris00/L-BFGS-ocaml"
+dev-repo: "git+https://github.com/Chris00/L-BFGS-ocaml.git"
+bug-reports: "https://github.com/Chris00/L-BFGS-ocaml/issues"
+doc: "https://Chris00.github.io/L-BFGS-ocaml/doc"
+tags: [ "science" "numerics" "optimization" "minimization" "maximization"
+        "bound-constrained" "large-scale" ]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "@doc"] {with-doc}
```
Comment (kit-ty-kate) [inline packages/lbfgs/lbfgs.0.9.3/opam]:
```suggestion
  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
```

File: packages/lbfgs/lbfgs.0.9.3/opam (hunk 1 above)
Comment (Chris00) [reply packages/lbfgs/lbfgs.0.9.3/opam]: Right — thanks!

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**